src/components/VoteCounter.js

```diff
@@ -27,7 +35,7 @@ export default function VoteCounter() {
           },
         }}
       >
-        นับแล้ว
+        <MobileVoteCounter totalCount={totalCount} />
```

Comment:
Let’s use JS-based responsive instead (`<Responsive>`) because this part of information depends on latest data.

https://github.com/codeforthailand/election-live#responsive-design